ocamlrep_ocamlpool/test/ocamlpool_test.rs

```diff
@@ -60,8 +69,12 @@ mod tests {
             "-cclib",
             "-locamlrep_ocamlpool",
         ]);
+        link_cmd.current_dir(tmp_dir.path());
         assert_eq!(run(link_cmd).map_err(fmt_exit_status_err), Ok(()));
-        let ocamlpool_test_cmd = sh_cmd(&["-c", "./ocamlpool_test"]);
+        let ocamlpool_test_cmd =
```

Comment:
At least this one is called so should it be marked as deadcode ?
Also, do we need to `include!` the files or can we depend on them "normally" like any other module ?